src/main/thrift/tag.thrift

```diff
@@ -74,6 +74,12 @@ struct PodcastMetadata {
 
     /** The Pocket Casts url for the podcast **/
     14: optional string pocketCastsUrl
+
+    /** Whether or not episodic art should be enabled */
+    15: optional bool episodicArtEnabled;
```

Comment:
I would like to make this `required` but I'm guessing we can't for backwards compatibility reasons?